### PR TITLE
metrics: Add common functions to the common script

### LINKED ISSUE #7617: metrics: Add common functions to the common.sh script

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the collect results function to the common metrics script.

Fixes #7617

### CODE REVIEW

File: tests/metrics/lib/common.bash [hunk 1]
```diff
@@ -361,3 +361,39 @@ function wait_ksm_settle()
 	done
 	info "Timed out after ${1}s waiting for KSM to settle"
 }
+
+function collect_results() {
+	WORKLOAD=${1}
+	[[ -z ${WORKLOAD} ]] && die "Container workload is missing"
+
+	local tasks_running=("${containers[@]}")
```
Comment (dborquez) [inline tests/metrics/lib/common.bash:369]: @GabyCT I think you should pass the variable `containers[@]` to this function.

**Review by dborquez — CHANGES_REQUESTED** [review]
LGTM in general.

**Review by dborquez — APPROVED** [review]
@GabyCT LGTM, thank you

**Review by cmaf — APPROVED** [review]
thanks @GabyCT

**Pull request merged**